### DISCUSSION TITLE
Imbedding buyer information into an STL that is being sold.

### DISCUSSION THREAD

**Discussion by rcamp5656** [community/community#4106] [Other Feature Feedback, Questions, & Ideas]
Hi: 

I was wondering if there was a way to imbed information as possibly comments into an STL that would identify a buyer of any STLs that are being sold. I am sure that there are users out there that would love to do this so that they could, through a plugin identify or code information as to who is buying that piece, where they are located or whatever other information that a seller would love to have imbedded into an STL.

I am a programmer but this is so far beyond me , if I knew how an STL worked right down to the basics I could write a standalone program in Quick Basic 64 and come up with something good. All I need to know is how to put in a comment into an STL and have it not effect printing. 

If anyone has any ideas on doing this , please get back to me. Russ Campbell

**Comment by Levi-Lesches** [comment]
> information... embedded into an STL. 

Seems like you're talking about modifying the file itself -- I don't think that's really an issue for the GitHub teams. These discussions are primarily about fixing bugs in and adding features to GitHub. Maybe you can take to StackOverflow.com? You can get help from people who know a lot about STL files and can tell you if this is possible.
